client/src/views/components/CreateProductSlide/CreateProductFooter/CreateProductFooter.js

```diff
@@ -0,0 +1,16 @@
+import { LOCATION_ICON } from '../../../../constants/imagePath.js';
+
+export default function CreateProductFooter({ $target, town }) {
+  this.$CreateFooter = document.createElement('div');
+  this.$CreateFooter.className = 'CreateProduct__footer';
+  $target.appendChild(this.$CreateFooter);
+
+  this.render = () => {
+    this.$CreateFooter.innerHTML = `
+      <img class="CreateProduct__footer__icon" src="${LOCATION_ICON}" />
```

Comment:
따옴표 빼도 실행이 될것 같습니다...! 아닌가요?

```suggestion
      <img class="CreateProduct__footer__icon" src=${LOCATION_ICON} />
```

Reply:
맞습니다 ㅠㅠ 따옴표가 없어도 상수 자체가 String으로 받아오기 때문에 따옴표를 빼도록 할게요!